<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_FontChooser">
        <property name="text">
         <string>Choose your font:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Reload">
        <property name="toolTip">
         <string>To reload the font list.</string>
        </property>
        <property name="text">
         <string>ðŸ”ƒ Reload Font List</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="cbx_FontChooser">
      <property name="toolTip">
       <string>Current Font used to print Text.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_ResetText">
      <property name="toolTip">
       <string>Clear text zone and add the defaut one.</string>
      </property>
      <property name="text">
       <string>Reset to default Text</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="te_Text"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
